railties/lib/rails/generators/app_base.rb

```diff
@@ -340,7 +340,15 @@ def spring_install?
       end
 
       def run_bundle
-        bundle_command('install') if bundle_install?
+        # Most of us have the CPUs; so let's parallelize `bundle install`
```

Comment:
@yuki24 I see what you are saying. I thought you were arguing the opposite, haha! My main motivation for using physical core number was to accommodate to older machines that do not do a ton of pipelining. That way most everyone gets a boost and we don't bother users with older machines with too much context switching.